site_root/input/resources/examples/Task-example-task-id.json

```diff
@@ -0,0 +1,34 @@
+{
+  "resourceType": "Task",
+  "id": "example-task-id",
+  "status": "completed",
```

Comment:
Could you add the following after `"id"`:

```json
  "identifier": [
    {
      "system": "urn:ncpi:unique-string",
      "value": "example-task-id"
    }
  ],
```

Reply:
resolved.